projects/ngx-interactive-paycard-demo/src/app/app.component.ts

```diff
@@ -1,4 +1,7 @@
 import { Component } from '@angular/core';
+import {CardLabel, FormLabel} from 'ngx-interactive-paycard-lib';
```

Comment:
Could you please add a whitespace before `CardLabel` and after `Formlabel` to follow the conventions. (If you use VS Code, `ALT+SHIFT+F` does it with the default settings.)